server/views/icons/other/cc_zero.svg

```diff
@@ -0,0 +1,19 @@
+<svg viewBox="-0.5 0.5 64 64">
+<g>

```

Comment:
Could we have this in the SVG optimiser?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/optimise-svgs.js